src/api-requests/get-all-search-content.ts

```diff
@@ -0,0 +1,13 @@
+import { SearchItem } from '../components/search/types';
+const searchEndpoint =
+    'https://data.mongodb-api.com/app/devhub-search-service-fldiy/endpoint/search_devcenter';
+
+// Should find a way to cache this response. Only use it in a few places for search filters but still.
```

Comment:
I think it might be worth capturing this in a Jira ticket to be worked on post-launch.